core/core.mk

```diff
@@ -94,6 +94,12 @@ libdir = core/lib/libfdt
 include mk/lib.mk
 endif
 
+ifeq ($(CFG_ZLIB),y)
```

Comment:
I guess when `CFG_ZLIB` isn't set `early_ta.c` won't compile?
Is it possible to use early TA feature without compression support?